go/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.14
+FROM golang:1.17-buster
   WORKDIR /home/app/repo
   COPY . /home/app/repo
 
-  ENV CLIENT_ID=
-  ENV CLIENT_SECRET=
-  ENV REDIRECT_URL=
+  ENV CLIENT_ID=7a8587d5fc8151cabf180a6a84fdf521251813ca204a2d4cf97f4072cb2d4585
```

Comment:
These secrets have been leaked and will need to be removed and rotated. Need to ensure these are no longer valid credentials before proceeding.

Reply:
These are local dev credentials from when I last used the repo so no risk, I'll wipe them locally and remove from here.

good spot.